packages/smooth_app/lib/pages/preferences/user_preferences_contribute.dart

```diff
@@ -123,16 +123,23 @@ class UserPreferencesContribute extends AbstractUserPreferences {
                       context: this.context,
                     );
                   },
-                  child: Text(
-                    appLocalizations.contribute_improve_ProductsToBeCompleted,
+                  child: Container(
+                    padding:
+                        const EdgeInsets.symmetric(horizontal: 8, vertical: 5),
```

Comment:
I believe there are constants already declared, if I remember `MEDIUM_SPACE`, like these, instead of hardcoding `8, 5, and 23` it would make sense to use a constant or maybe in case you need a new one declare that as well

Reply:
So Should I replace constant values ? and use **MEDIUM_SPACE** instead or something else

Reply:
Yes @Akashsri3bi

Reply:
okay great !

Reply:
While thinking about it again, using a `SmoothActionButton` would simplify this even more @Akashsri3bi, the thinking is to just have one place to change values in one place if we decide to change something on the theme

Reply:
So instead of container , I should use a SmoothActionButton so change from one place can be reflected everywhere is that what you want to imply @M123-dev 🙄 ?

Reply:
If you see, we use a SmoothActionButton for the purpose of buttons here, which have it's own properties and decorations,
So if you use that class and change the properties there itself, it would make more sense.


I hope i sounded clear, if you need any help let me know

Reply:
Oh Okay I am getting your point I will see to it , If I run into problems I will let you know Thanks @AshAman999 !

Reply:
@AshAman999 @M123-dev while looking through it there are few things , so previously also we are using **SmoothActionButton** as our positive action for our dialog which is 'okay' , now if you want to use SmoothActionButton for **products to be completed** button then we can't assign it to list type widget which is inside our column , Best way what I think can be inkwell I will correct it to use constant declared spacing . And if we are making  **products to be completed** a SmoothActionButton then we would have to make it a widget type which possibly would disturb parts where this was used before . What are your suggestions !

Reply:
@Akashsri3bi 
There is a negative action as well, right? did you try using that 

Also perhaps you can remove the   `products to be completed ` from the column and put it next to the positive action as a negative action, and since the negative action is being used at multiple places, it would be a bit easier to have consistency with looks throughout the whole app. I hope I things a bit clear here

Reply:
Ohh , Yes Thanks I got your point !

Reply:
That's the golden tests in action for now, can you send the screenshot page in light and dark theme

Reply:
![Screenshot_2022-10-26-23-07-53-90 1](https://user-images.githubusercontent.com/71835085/198097799-513c9eb9-b869-4ed6-9467-75b49602fc39.png)

![Screenshot_2022-10-26-23-07-03-66 1](https://user-images.githubusercontent.com/71835085/198098001-b3dae1a4-cb74-4f64-ba34-130c37fd4c05.png)

Sure @AshAman999

Reply:
Now the contents aren't readable, 
Can we tweak the negative action button's properties to make it readable,

Looking after this, we were much better with the early version you did, 
Any suggestions here @M123-dev >?

Reply:
I tried to do some changes in class property but It does not works , What would you recommend ?